services/app-api/serverless.yml

```
@@ -204,6 +207,14 @@ functions:
           method: delete
           cors: true
           authorizer: ${self:custom.authValue.${self:custom.stage}, ""}
+  getRate:
+    handler: handlers/rate/get.getRate
+    events:
+      - http:
+          path: rate/{state}/{year}/{coreSet}/{measure}/get
+          method: get
```

The `/get` here is in line with the QMR convention, but it is redundant with the `method`, and the convention has already been bucked by the banner endpoints. I'd recommend changing it, but I'd also recommend changing all the other endpoints... once again, a today vs. someday tradeoff to make.

```suggestion
          path: rate/{state}/{year}/{coreSet}/{measure}
          method: get
```

---

added it to a ticket as a global issue to look into fixing for all the paths